.github/workflows/rust.yml

```diff
@@ -10,13 +10,40 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
-  unit_tests:
+  build:
```

Comment:
job name `build` here is duplicated from step name `Build`. Would suggest to change job name to `ci` for continuous integration